src/website/src/pages/ec/components/site-wide/site-header/docs/code.mdx

```diff
@@ -18,6 +18,7 @@ import {
   selectedKind="components-site-wide-site-header"
   selectedStory="core"
   showFrame
+  frameHeight="700"
```

Comment:
i guess that you added this to fix some issue you had without it, but the experience in the website is pretty bad, closing the popover generates a messy and super quick collapse and expand, generally we disable the autoResize when we pre-define the size of the iframe, like this:
```
frameHeight="700"
disableAutoResize
```

Reply:
thanks, that's indeed better